terraform-providers-mirror/.terraform.lock.hcl

```diff
@@ -0,0 +1,20 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
This lock file might not be necessary if this is used for TF 0.13.x examples

Reply:
This is a very valid point!